plasma_framework/contracts/src/vaults/predicates/IErc20DepositVerifier.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import {PaymentTransactionModel as DepositTx} from "../../transactions/PaymentTransactionModel.sol";
```

Comment:
we cannot hardcode our tx model dependency in interface, otherwise we are in the same situation that we cannot upgrade the tx type for deposit. Then there would be really less benefit to put an interface on the verifier.

That is why my previous interface for ETH was taking raw bytes as input, and it is the implementation decide which tx model to use to decode.

Reply:
Understood, thanks 👍